src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actor")
```

Comment:
```suggestion
@Table(name = "actors")
```

read naming convention for databases 
table name should be plural